src/Tickets/Seating/Service/Service.php

```diff
@@ -234,10 +234,13 @@ public function get_frontend_url( string $path = '' ): string {
 	 * @return string The URL to load the Maps create and edit page.
 	 */
 	public function get_map_edit_url( string $token, string $map_id ): string {
+		$associated_posts = $this->layouts->get_associated_posts_by_id( $layout_id);
```

Comment:
This is the map edit url. We need to change the layout edit URL here:

https://github.com/the-events-calendar/event-tickets/blob/768e2a6b468a9b5a3d5a02662fc191230ad5fe99/src/Tickets/Seating/Service/Service.php#L280

Reply:
Replace the args with this:

```
return add_query_arg(
			[
				'token'    => urlencode( $token ),
				'layoutId' => urlencode( $layout_id ),
				'associatedEvents' => Layouts::get_associated_posts_by_id( $layout_id ),
			],
			$this->get_frontend_url( '/embed/seat-layout/' )
		);